src/components/categoriesBar/CategeriesBar.jsx

```diff
@@ -0,0 +1,7 @@
+import "./_categoriesBar.scss";
```

Comment:
filename typo here

Reply:
`src/components/categoriesBar/CategeriesBar.jsx`

Reply:
changing it